src/backend/joanie/remote_endpoints_urls.py

```diff
@@ -0,0 +1,15 @@
+"""
+API routes exposed for server to server. Requires a specific token to request.
+"""
+from django.conf import settings
+from django.urls import path
+
+from joanie.core.api.remote_endpoints import enrollments_and_orders_on_course_run
+
+urlpatterns = [
+    path(
+        f"api/{settings.API_VERSION}/enrollments-and-orders-on-course-run/",
```

Comment:
Sorry for not pointing this earlier, I just think about it.
What do you think about renaming this route
```suggestion
        f"api/{settings.API_VERSION}/course-run-metrics/",
```
This way, we could add more data as needed, without changing the route name.
Also, it would match the called method `get_course_run_metrics`